examples/main.tf

```diff
@@ -141,6 +141,15 @@ resource "instaclustr_vpc_peering_gcp" "example_vpc_peering" {
   cluster_id = "${instaclustr_cluster.gcp_example.id}"
 }
 
+resource "instaclustr_vpc_peering_azure" "example_vpc_peering" {
```

Comment:
Add some spaces between the equals signs in the below example, the whitespace will make it look a bit cleaner